tests/framework/PostProcessors/ParetoFrontierPostProcessor/test_paretoFrontierWithCostLimit.xml

```diff
@@ -0,0 +1,62 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/ParetoFrontierPostProcessor.ParetoFrontier</name>
```

Comment:
change ``ParetoFrontier`` to ``ParetoFrontierWithCostLimit``

Reply:
edited